### DISCUSSION TITLE
What method should I call when the stripe client payment status is past due?

### DISCUSSION THREAD

**Discussion by klaus-trabalhando** [pay-rails/pay#809] [Q&A]
I'm trying to find a documentation about what to do when the user status is past due.
The user wants to renew his yearly plan (using stripe).
Should I let the app call `user.payment_processor.checkout` again or should I call another method when user has a past due subscription?

I appreciate any help

**Comment by excid3** [comment]
You would update to a valid payment method and pay the latest invoice on the subscription (if I remember correctly).

You can lookup the subscription and call `pay_open_invoices`

Or after updating the payment method, you can attempt to pay any unpaid subscriptions.
`payment_processor.subscriptions.unpaid.each(&:pay_open_invoices)`
